diplom.py

```diff
@@ -19,7 +27,7 @@ def mean_average_precision(predicted_probability_distribution,
         classes_by_prob = []
         for i in xrange(0, len(list_of_classes)):
             classes_by_prob.append([row[i], list_of_classes[i]])
-            classes_by_prob.sort(reverse=True)    
+        classes_by_prob.sort(reverse=True)    
         for j in range (0, 5):
```

Comment:
I don't understand what did u mean by this:
 make this min(5, len(classes_by_prob))